fbw-common/src/systems/datalink/router/src/webinterfaces/HoppieConnector.ts

```diff
@@ -388,6 +388,8 @@ export class HoppieConnector {
     }
 
     public static pollInterval(): number {
```

Comment:
Maybe a good idea to add a warning that the value is not static anymore..
```suggestion
    /**
     * Gets the interval to poll the Hoppie API in milliseconds.
     * Warning: This will return a different random time on each invocation!
     * @returns The polling interval in milliseconds)
     */
    public static pollInterval(): number {
```

Reply:
Agreed, good idea.